go/types/indexed_sequences.go

```diff
@@ -83,7 +83,7 @@ func orderedKeyFromSum(msd []metaTuple) orderedKey {
 
 // loads the set of leaf nodes which contain the items [startIdx -> endIdx).
```

Comment:
Can you make this a valid GoDoc comment?

Reply:
done